upgrade-tests/src/test/java/io/zeebe/test/UpgradeTest.java

```diff
@@ -101,6 +101,42 @@
                 .afterUpgrade(UpgradeTest::publishMessage)
                 .done()
           },
+          // TODO (saig0): enable the test case when upgrading from 0.25.0
+          // - an upgrade from 0.24.0 is not possible because a bug fix (#4959) causes an issue in
+          // the reprocessing (#5268)
+          //            {
+          //            "message event sub-process",
+          //            scenario()
+          //                .deployWorkflow(
+          //                    Bpmn.createExecutableProcess(PROCESS_ID)
+          //                        .eventSubProcess(
+          //                            "event-subprocess",
+          //                            eventSubProcess ->
+          //                                eventSubProcess
+          //                                    .startEvent()
+          //                                    .message(
+          //                                        m ->
+          // m.name(MESSAGE).zeebeCorrelationKeyExpression("key"))
+          //                                    .interrupting(false)
+          //                                    .endEvent())
+          //                        .startEvent()
+          //                        .serviceTask(TASK, t -> t.zeebeJobType(TASK))
+          //                        .endEvent()
+          //                        .done())
+          //                .createInstance(Map.of("key", "123"))
+          //                .beforeUpgrade(
+          //                    state -> {
+          //                      publishMessage(state, -1L, -1L);
+          //
+          //                      TestUtil.waitUntil(
+          //                          () -> state.hasElementInState("event-subprocess",
+          // "ELEMENT_COMPLETED"));
+          //
+          //                      return activateJob(state);
+          //                    })
+          //                .afterUpgrade(UpgradeTest::completeJob)
+          //                .done()
+          //          },
```

Comment:
I added the test case for the next release. I was also useful to verify that the issue is detected :)

Reply:
How do we remind ourselves to enable this? :)